ShowsLibrary/ShowsLibrary/Class1.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using HRLibrary;
+using ShowsLibrary;
+
+namespace HRLibrary
+{
+    public class Showw
+    {
+        public string Name { get; set; }
+        public string Surname { get; set; }
+        public readonly DateTime Birthday;
+        public readonly Gender Gender;
+        private string v1;
+        private string v2;
+        private string v3;
+        private Frequency daily;
+        private string v4;
```

Comment:
Закрытые поля `v1`, ..., ` v4` не нужны. Вы их не используете.
У вас используются одноименные параметры конструктора, но к этим полям они не имеют никакого отношения.
```suggestion
        private Frequency daily;
```